### DISCUSSION TITLE
Saving file on a path

### DISCUSSION THREAD

**Discussion by SanaWaseem** [SixLabors/ImageSharp.Web#273] [Q&A] [answered]
Can someone here answer my query here: https://stackoverflow.com/questions/72572835/save-an-image-with-a-new-file-name-imagesharp/72587592#72587592
I cannot find the overload where the complete path is given, for now I'm not even sure where the file is being saved. I need to save it on a location different than the web root image path.

**Comment by JimBobSquarePants** [comment, marked as answer]
This is the wrong library. This QA is for the ASP.NET Core Middleware.

**Reply by SanaWaseem** [reply]
I need to use ImageSharp.web as I am using .net core. Please suggest how can we give filepath here as my file is saving on a ghost path and I am unable to figure it out.

**Reply by JimBobSquarePants** [reply]
No you don’t. You need ImageSharp.

**Reply by SanaWaseem** [reply]
Can we use ImageSharp in net core as well ?

**Reply by JimBobSquarePants** [reply]
Yes.
